components/shared/NavComponent.js

```diff
@@ -69,12 +69,12 @@ export default function NavComponent(props) {
 
               return (
                 <div key={page.route} className={`flex-grow-1 mx-0 my-auto`}>
-                  <Link href={page.route} passHref>
-                    <Nav.Link
-                      className={`text-primary text-center px-3 py-1 navbar-menu-link ${fontWeightClass}`}
-                    >
-                      {page.label}
-                    </Nav.Link>
+                  <Link
+                    href={page.route}
+                    passHref
+                    className={`text-primary text-center px-3 py-1 navbar-menu-link ${fontWeightClass}`}
+                  >
+                    {page.label}
                   </Link>
```

Comment:
This was a hydration error where our components ended up rendering one anchor inside another which is not allowed.